src/conways.rs

```diff
@@ -29,6 +30,18 @@ impl Conways {
         }
     }
 
+    pub fn set_dead(&mut self, row: usize, column: usize) {
+        if let Some(row_grid) = self.world.get_mut(row) {
+            if let Some(cell) = row_grid.get_mut(column) {
+                *cell = CellState::Dead;
+            }
+        }
+    }
```

Comment:
Instead of having a set_dead and set_alive, you could have a function with the following signature `set_cell(&mut self, row:usize,col:usize,state:CellState)`

Reply:
I wanted it to be encapsulated, the user of the module doesn't know the internal representation, the enum of `CellState` is private. What do you think? is it too much to have separate functions?

Reply:
It is a good point, for me it is not wrong to expose the `CellState` enum